src/elementPlugins/shortText/src/ApplicationView.tsx

```diff
@@ -7,52 +7,25 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
   onUpdate,
   initialValue,
   isEditable,
-  isRequired,
-  allResponses,
-  evaluator,
+  currentResponse,
+  validationState,
+  onSave,
 }) => {
-  const [validationMessageDisplay, setValidationMessageDisplay] = useState('')
   const [value, setValue] = useState(initialValue?.text)
-  const [isValid, setIsValid] = useState<boolean>()
-  const [responses, setResponses] = useState({
-    thisResponse: undefined,
-    ...allResponses,
-  })
-  const {
-    validation: validationExpression,
-    validationMessage,
-    placeholder,
-    maskedInput,
-    label,
-  } = templateElement?.parameters
+  const { placeholder, maskedInput, label } = templateElement?.parameters
 
   useEffect(() => {
-    // Do validation, setIsValid
-    if (validationExpression && responses.thisResponse !== undefined) {
-      evaluator(validationExpression, { objects: [responses], APIfetch: fetch })
-        .then((result: boolean) => {
-          setIsValid(result)
-        })
-        .catch((err: any) => console.log(err))
-    } else setIsValid(true)
-  }, [responses])
-
-  useEffect(() => {
-    setResponses({ thisResponse: value, ...allResponses })
-  }, [value])
-
-  useEffect(() => {
-    if (isValid) setValidationMessageDisplay('')
-    else setValidationMessageDisplay(validationMessage)
-  }, [isValid])
+    onUpdate(value)
+  }, [])
 
   function handleChange(e: any) {
+    onUpdate(e.target.value)
     setValue(e.target.value)
   }
```

Comment:
This is so different from what is in the `dropdownChoice/TemplateView.tsx`:

```
const onChange = (_: any, { value }: InputOnChangeData) => {
    setPlaceholder(value)
    onUpdate({ placeholder: value })
  }
```

Should we keep one pattern? I would rather the version from `dropdownChoice`

Reply:
Similar to the one below: `handleLoseFocus`

Reply:
We didn't touch TemplateView -- what we've done in ApplicationView is intended to be the standard.

Reply:
I think 🤷‍♂️